.github/workflows/e2e-arm64.yaml

```diff
@@ -0,0 +1,31 @@
+name: E2E-arm64
```

Comment:
Instead of adding a separate workflow, can you add arm64 as a job to existing "E2E" workflow?

Reply:
It doesn't work because the ARM machines have different labels. FYI. https://github.com/etcd-io/etcd/pull/15172#discussion_r1084600047

Probably we can add a template to be shared by both the existing E2E workflow and ARM64. But I don't think it's a big deal, It's OK to leave it as it's for now.

Reply:
Labels are per job not per forkflow.
```
jobs:
  test:
    runs-on: [Linux, ARM64]
```
You can change it to:

```
jobs:
  amd64:
    runs-on: ubuntu-latest
    ...
  arm64:
    runs-on: [Linux, ARM64]
```